firefox-ios/Shared/Strings.swift

```diff
@@ -217,10 +217,10 @@ extension String {
                     value: "Save sites as you browse. We’ll also grab bookmarks from other synced devices.",
                     comment: "The body text for the placeholder screen shown when there are no saved bookmarks, located at the root level of the bookmarks panel within the libray modal")
                 public static let ButtonTitle = MZLocalizedString(
-                    key: "Bookmarks.EmptyState.Root.ButtonTitle.v135",
+                    key: "Bookmarks.EmptyState.Root.ButtonTitle.v135.v2",
```

Comment:
Was that string exported already to l10n?

Reply:
Yep - https://github.com/mozilla-l10n/firefoxios-l10n/blob/564abb4d187ffa32e1b6081fbeb8dc8afbf8573f/en-US/firefox-ios.xliff#L4083-L4087

Reply:
Mhh, I think we should be careful then so we don't loose translations? Is the `EmptyStateView` protected behind a feature flag? I think it was added as part of v134 release train. If it's not protected behind a feature flag then the next string import from l10n into Beta/Release builds will remove all translated strings for "Sign in to Sync". 

That means we should use the [`oldStrings` struct](https://github.com/mozilla-mobile/firefox-ios/wiki/How-to-add-and-modify-Strings#oldstrings-struct) instead. Also, I would change the key to `.v136` instead of adding a `.v2` in that case.

Reply:
I believe this string went in with v135. It is behind a feature flag. I was under the impression we wanted re-translations with the new context?

Reply:
But I think you are correct that adding a `v2` here is unnecessary, and we can just go with `v136` since the string is not in use.

Reply:
If it's protected behind feature flag, then all good 😄